src/main/java/io/appium/java_client/android/AndroidMobileOptions.java

```diff
@@ -0,0 +1,1716 @@
+    public AndroidMobileOptions setSkipUnlock() {
+        return setSkipUnlock(true);
+    }
+
+    /**
+     * Set whether to skip unlock during session creation.
+     *
+     * @param bool when true skips unlock during session creation.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#SKIP_UNLOCK
+     */
+    public AndroidMobileOptions setSkipUnlock(boolean bool) {
+        return amend(AndroidMobileCapabilityType.SKIP_UNLOCK, bool);
+    }
+
+    /**
+     * Get whether to skip unlock during session creation.
+     *
+     * @return true if unlock during session creation is skipped.
+     * @see AndroidMobileCapabilityType#SKIP_UNLOCK
+     */
+    public boolean doesSkipUnlock() {
+        return (boolean) getCapability(AndroidMobileCapabilityType.SKIP_UNLOCK);
+    }
+
+    /**
+     * Set the port used to connect to the driver.
+     *
+     * @param port is the port used to connect to the driver.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#SYSTEM_PORT
+     */
+    public AndroidMobileOptions setSystemPort(String port) {
+        return amend(AndroidMobileCapabilityType.SYSTEM_PORT, port);
+    }
+
+    /**
+     * Get the port used to connect to the driver.
+     *
+     * @return String of the port used to connect to the driver.
+     * @see AndroidMobileCapabilityType#SYSTEM_PORT
+     */
+    public String getSystemPort() {
+        return (String) getCapability(AndroidMobileCapabilityType.SYSTEM_PORT);
+    }
+
+    /**
+     * Set the app to enable Unicode input.
+     *
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#UNICODE_KEYBOARD
+     */
+    public AndroidMobileOptions setUnicodeKeyboard() {
+        return setUnicodeKeyboard(true);
+    }
+
+    /**
+     * Set whether to enable Unicode input.
+     *
+     * @param bool when true enables Unicode input.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#UNICODE_KEYBOARD
+     */
+    public AndroidMobileOptions setUnicodeKeyboard(boolean bool) {
+        return amend(AndroidMobileCapabilityType.UNICODE_KEYBOARD, bool);
+    }
+
+    /**
+     * Get whether to enable Unicode input.
+     *
+     * @return true if Unicode input is enabled.
+     * @see AndroidMobileCapabilityType#UNICODE_KEYBOARD
+     */
+    public boolean isUnicodeKeyboard() {
+        return (boolean) getCapability(AndroidMobileCapabilityType.UNICODE_KEYBOARD);
+    }
+
+    /**
+     * Set the list of packages to uninstall before installing apks.
+     *
+     * @param packages is the list of packages to uninstall before installing apks.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#UNINSTALL_OTHER_PACKAGES
+     */
+    public AndroidMobileOptions setUninstallOtherPackages(List<String> packages) {
+        return amend(AndroidMobileCapabilityType.UNINSTALL_OTHER_PACKAGES, packages);
```

Comment:
I assume this has to be a comma separated string: https://github.com/appium/appium-android-driver/blob/e2a7e4b5407b845ce7f21b8e4b0be0c2cc0fd01a/lib/desired-caps.js#L190

Reply:
Oh, that's an interesting file that I didn't know about. :)
We should update the documentation then, since it lists this as an option:
`e.g. "io.appium.example", ["io.appium.example1", "io.appium.example2"], '*'`

Reply:
https://github.com/appium/appium/pull/14133
1453a1a68bbe5b472568b94e8c4830430ce494d1